lang/cs/Org.Apache.REEF.Evaluator/Evaluator.cs

```diff
@@ -156,74 +111,22 @@ private static void AttachDebugger()
                     break;
                 }
 
-                logger.Log(Level.Info, "Evaluator in debug mode, waiting for debugger to be attached...");
+                _logger.Log(Level.Info, "Evaluator in debug mode, waiting for debugger to be attached...");
                 Thread.Sleep(2000);
```

Comment:
This is original code. I am not sure. I guess it is for loading symbols for debug